justfile

```diff
@@ -8,10 +8,10 @@ alias dog := docs-openapi-generate
 localdev-cleandb:
     rm ./localdev/storage/db/ingest.sqlite
 
-# Build a docker image locally and run it with docker-compose up
+# Build a docker image locally and run it with docker compose up
 build-and-run-docker:
     export DOCKER_BUILDKIT=1; sbt Docker/publishLocal
-    docker-compose up -d
+    docker compose up -d
     docker compose logs -f
```

Comment:
I did this just for consistency. Line 15 calls `docker compose` already, so I changed the other occurrences, too